src/main/java/seedu/duke/util/Message.java

```diff
@@ -31,24 +31,26 @@
     public static final String MESSAGE_INCOMPLETE_UNDATED_TASKLIST = "\nIncomplete undated tasks:";
     public static final String MESSAGE_INCOMPLETE_DATED_TASKLIST = "\nIncomplete dated tasks:";
     public static final String MESSAGE_SUMMARY_PRINTED = "Your Task Summary\n";
-    public static final String MESSAGE_HELP =
-              "Command                                       Function\n"
-            + "help ---------------------------------------- View command list\n"
-            + "add -t <task desc.> [-by] ------------------- Add task\n"
-            + "add -m <module> ----------------------------- Add module\n"
-            + "edit -t <task index> <new task desc.> ------- Edit a task's description\n"
-            + "edit -m <module code> <new module code> ----- Edit a module\n"
-            + "del -t <task index> ------------------------- Delete a task\n"
-            + "del -m <module index> ----------------------- Delete a module\n"
-            + "list -t ------------------------------------- List all tasks\n"
-            + "list -m ------------------------------------- List all modules\n"
-            + "done <task index> --------------------------- Mark task as done\n"
-            + "timetable -day ------------------------------ View the timetable for today\n"
-            + "timetable -week ----------------------------- View the timetable for the week\n"
-            + "timetable -add <module> <day> <start time> \n"
-            + "<end time> <lesson type> <repeat> ----------- Add a lesson to the timetable\n"
-            + "timetable -del <day> <lesson index> --------- Delete a lesson from the timetable\n"
-            + "bye ----------------------------------------- Exit RaVi\n";
+    public static final String MESSAGE_COMMAND_LIST =
+            "Command                                        Function\n"
```

Comment:
All the `<...>` fields should be named the same way as in the UG. Refer to the above comments to see examples.